ppr-ui/src/views/newMhrRegistration/SubmittingParty.vue

```diff
@@ -193,20 +193,20 @@ export default defineComponent({
         const validateDocId: MhrDocIdResponseIF = await validateDocumentID(localState.documentId)
         localState.isUniqueDocId = !validateDocId.exists && validateDocId.valid
         localState.displayDocIdError = !localState.isUniqueDocId
+        localState.isVerifiedDocId
+          ? setValidation(MhrSectVal.SUBMITTING_PARTY_VALID, MhrCompVal.DOC_ID_VALID, true)
+          : setValidation(MhrSectVal.SUBMITTING_PARTY_VALID, MhrCompVal.DOC_ID_VALID, false)
```

Comment:
this could be simplified to `setValidation(MhrSectVal.SUBMITTING_PARTY_VALID, MhrCompVal.DOC_ID_VALID, localState.isVerifiedDocId` i think

Reply:
You're right, I'll do that thanks!